apps/api/src/domain/models/audio-item/commands/transcripts/import-line-items-to-transcript/import-line-items-to-transcript.command-integration.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { Video } from '../../../entities/video.entity';
+import { ImportLineItemsToTranscript } from './import-line-items-to-transcript.command';
+
+const commandType = `IMPORT_LINE_ITEMS_TO_TRANSCRIPT`;
+
+const testDatabaseName = generateDatabaseNameForTestSuite();
+
+const systemUserId = buildDummyUuid();
+
+const existingAudioItem = getValidAggregateInstanceForTest(AggregateType.audioItem);
+
+const existingVideo = getValidAggregateInstanceForTest(AggregateType.video);
+
+const validAudiovisualItems = [existingAudioItem, existingVideo];
+
+describe(commandType, () => {
+    let testRepositoryProvider: TestRepositoryProvider;
+
+    let commandHandlerService: CommandHandlerService;
+
+    let app: INestApplication;
+
+    let idManager: IIdManager;
+
+    let assertionHelperDependencies: CommandAssertionDependencies;
+
+    beforeAll(async () => {
+        ({ testRepositoryProvider, commandHandlerService, idManager, app } =
+            await setUpIntegrationTest({
+                ARANGO_DB_NAME: testDatabaseName,
+            }));
+
+        assertionHelperDependencies = {
+            testRepositoryProvider,
+            commandHandlerService,
+            idManager,
+        };
+    });
+
+    afterAll(async () => {
+        await app.close();
+    });
+
+    beforeEach(async () => {
+        await testRepositoryProvider.testSetup();
+    });
+
+    afterEach(async () => {
+        await testRepositoryProvider.testTeardown();
+    });
+
+    validAudiovisualItems.forEach((validInstance) => {
+        describe(`when transcribing a ${formatAggregateType(validInstance.type)}`, () => {
+            const [startTime, endTime] = validInstance.getTimeBounds();
+
+            const audioLength = endTime - startTime;
+
+            const numberOfTimestampsToGenerate = 10;
+
+            const epsilon = 0.0001;
+
+            const allTimestamps = Array(numberOfTimestampsToGenerate)
+                .fill(0)
+                .map((_, index) => [
+                    startTime + epsilon + (index * audioLength) / numberOfTimestampsToGenerate,
+                    startTime +
+                        ((index + 1) * audioLength) / numberOfTimestampsToGenerate -
+                        epsilon,
+                ]);
+
+            const dummyText = 'bla bla bla';
+
+            const languageCode = LanguageCode.Chilcotin;
+
+            const multilingualText = buildMultilingualTextWithSingleItem(dummyText, languageCode);
+
+            const buildValidCommandFSA = (
+                validInstance: AudioItem | Video
+            ): FluxStandardAction<ImportLineItemsToTranscript> => ({
+                type: commandType,
+                payload: {
+                    aggregateCompositeIdentifier:
+                        validInstance.getCompositeIdentifier() as ResourceCompositeIdentifier<
+                            typeof ResourceType.audioItem | typeof ResourceType.video
+                        >,
+                    lineItems: [
```

Comment:
Then when you get to this point, you can use your above helper while mapping over all time stamps as follows.
```ts
lineItems: allTimeStamps.map(
  ([inPointMilliseconds,outPointMilliseconds]) => ({ 
    inPointMilliseconds,
    outPointMilliseconds,
    text: buildDummyText([inPointMilliseconds,outPointMilliseconds]),
    languageCode,
    speakerInitials: validInstance.transcript.participant[0].initials
})
)
```

Reply:
has been done